client/src/elm/Pages/NCD/RecurrentActivity/Model.elm

```diff
@@ -0,0 +1,111 @@
+module Pages.NCD.RecurrentActivity.Model exposing (..)
+
+import Backend.Entities exposing (..)
+import Backend.Measurement.Model exposing (..)
+import Date exposing (Date)
+import EverySet exposing (EverySet)
+import Gizra.NominalDate exposing (NominalDate)
+import Measurement.Model
+    exposing
+        ( CreatinineResultForm
+        , LaboratoryTask
+        , LiverFunctionResultForm
+        , RandomBloodSugarResultForm
+        , UrineDipstickResultForm
+        , emptyCreatinineResultForm
+        , emptyLiverFunctionResultForm
+        , emptyRandomBloodSugarResultForm
+        , emptyUrineDipstickResultForm
+        )
+import Pages.NCD.RecurrentActivity.Types exposing (..)
+import Pages.Page exposing (Page)
+
+
+type Msg
+    = NoOp
+    | SetActivePage Page
+      -- Lab Results msgs
+    | SetActiveLabResultsTask LaboratoryTask
+    | SetProtein String
+    | SetPH String
+    | SetGlucose String
+    | SetLeukocytes String
+    | SetNitrite String
+    | SetUrobilinogen String
+    | SetHaemoglobin String
+    | SetKetone String
+    | SetBilirubin String
+    | SaveUrineDipstickResult PersonId (Maybe ( NCDUrineDipstickTestId, NCDUrineDipstickTest )) (Maybe LaboratoryTask)
+    | SetRandomBloodSugar String
+    | SaveRandomBloodSugarResult PersonId (Maybe ( NCDRandomBloodSugarTestId, NCDRandomBloodSugarTest )) (Maybe LaboratoryTask)
+    | SetCreatinineResult String
+    | SetUreaResult String
+    | SetNitorogenResult String
+    | SaveCreatinineResult PersonId (Maybe ( NCDCreatinineTestId, NCDCreatinineTest )) (Maybe LaboratoryTask)
+    | SetAltResult String
+    | SetAstResult String
+    | SaveLiverFunctionResult PersonId (Maybe ( NCDLiverFunctionTestId, NCDLiverFunctionTest )) (Maybe LaboratoryTask)
+      -- NextStepsMsgs
+      -- | SetActiveNextStepsTask NextStepsTask
+      -- | SetReferralBoolInput (Bool -> ReferralForm -> ReferralForm) Bool
+      -- | SetFacilityNonReferralReason (Maybe ReasonForNonReferral) ReferralFacility ReasonForNonReferral
+      -- | SaveSendToHC PersonId (Maybe ( NCDSendToHCId, NCDSendToHC )) (Maybe NextStepsTask)
+      -- | SetMedicationDistributionBoolInput (Bool -> MedicationDistributionForm -> MedicationDistributionForm) Bool
+      -- | SetMedicationDistributionAdministrationNote (Maybe AdministrationNote) MedicationDistributionSign AdministrationNote
+      -- | SetRecommendedTreatmentSign (List RecommendedTreatmentSign) RecommendedTreatmentSign
+      -- | SaveMedicationDistribution PersonId (Maybe ( NCDMedicationDistributionId, NCDMedicationDistribution )) (Maybe NextStepsTask)
+      -- | SetHealthEducationBoolInput (Bool -> HealthEducationForm -> HealthEducationForm) Bool
+      -- | SaveHealthEducation PersonId (Maybe ( NCDHealthEducationId, NCDHealthEducation )) (Maybe NextStepsTask)
```

Comment:
```suggestion
```

Shall we get rid of the commented out code or is it kind of a todo?

Reply:
Next steps related development is done at https://github.com/TIP-Global-Health/eheza-app/pull/456 and https://github.com/TIP-Global-Health/eheza-app/pull/459, All commented out code is handled there.